ming_sim/cli_backend.py

```
@@ -1198,12 +1198,14 @@ def classify_cli_action_intent(
         "判断本轮是否属于一个或多个政务动作，并抽出可从皇帝话与相关上下文直接确定的结构字段。"
         "单动作输出一个 JSON 对象，多动作输出 JSON 对象数组（无代码围栏、无多余字）：\n"
         + schema_obj + "\n"
-        "规则：确认优先于新动作；同一话语可同时含拟旨与任免时须同时输出"
-        "拟旨与任免候选（draft+appointment 并存），多候选不得因拟旨省略任免。"
-        "「拟旨如下」只是路由，不是动作类型。同一话语带拟旨前缀且为载荷式拨饷"
+        "规则：确认优先于新动作；每一道独立旨意只判一次颁布方式，同一动作类型不得重复候选。"
```

Ruff (0.16.3)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)